### ISSUE TITLE
SVG sprite being read out to Screen Readers on site enter

### ISSUE THREAD

**Issue by leandrasilver** [jkphl/svg-sprite#237]
On page enter, the SVG sprite is being read out to screen readers which may create confusion.

**Comment by charlestati** [comment]
Hi!

Did you try adding `aria-hidden="true"` to the sprite?

**Comment by tedw** [comment]
You could also try adding `visibility: hidden;` on the SVG or a wrapper `<div>`.

**Comment by rickydazla** [comment]
@charlestati could you (or anyone else) possibly provide some guidance on how would I go about adding `aria-hidden="true"` to the SVG? This is my config:

```
svg: {
  namespaceClassnames: false
},
mode: {
  css: false,
  symbol: {
    inline: true,
    sprite: 'icon-sprite.svg',
    dest: '.'
  }
}
```

**Comment by jkphl** [comment]
@rickydazla As @charlestati stated earlier, adding `aria-hidden="true"` to the sprite `svg` element or to a wrapping element would hide the sprite for screenreaders. Adding `aria-hidden` to a wrapping element would be outside of *svg-sprites* scope (you should be able to do this in your HTML markup that embeds the sprite). Alternatively, you can either use [`svg.rootAttributes`](https://github.com/jkphl/svg-sprite/blob/master/docs/configuration.md#sprite-svg-options) or a [global post-processing transformation](https://github.com/jkphl/svg-sprite/blob/master/docs/configuration.md#svg-sprite-customization) to add the attribute to the sprite's `svg` element. Please see the docs for additional hints.
